plotly/offline/offline.py

```diff
@@ -282,18 +295,20 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
 
     if image:
         if image not in __IMAGE_FORMATS:
+            img_n = len(__IMAGE_FORMATS)
             raise ValueError('The image parameter takes only the '
-                             'following format types: `png`, `jpeg`, '
-                             '`webp`, `svg`')
-        # if the check passes then download script injection will commence.
-        # Write script to download image of the plot
+                             'following format types: `{}`, `{}`, '
+                             '`{}`, `{}`'.format(*[__IMAGE_FORMATS[i]
+                                                 for i in range(img_n)]
+                                                 )
```

Comment:
;__; this will just break if you don't have enough `{}`, right?

Can you:

```
'This image parameter must be one of {}.'.format(__IMAGE_FORMATS)
```